app/models/pipeline_run.rb

```diff
@@ -368,6 +368,7 @@ def db_load_taxon_counts
     update_names
     # denormalize superkingdom_taxid into taxon_counts
     if multihit?
+      update_kingdoms
```

Comment:
Can we squash update_kingdoms & update_superkingdoms into one?

Reply:
done

Reply:
Thanks. I meant put it into a single SQL query though. Can't you just add another column/value pair to the SET clause? Am I missing something?